common/test/acceptance/pages/lms/course_nav.py

```diff
@@ -126,6 +126,7 @@ def go_to_sequential(self, sequential_title):
             # Convert the list index (starts at 0) to a CSS index (starts at 1)
             seq_css = "ol#sequence-list>li:nth-of-type({0})>a".format(seq_index + 1)
             self.q(css=seq_css).first.click()
+            self.wait_for_ajax()
```

Comment:
If you are not sure, you should be able to see the behavior in Firebug or Chrome developer tools...to see whether or not there is an ajax call after this click.